### ISSUE TITLE
expo-modules-core-migration

### ISSUE THREAD

**Issue by Bardiamist** [expo/fyi#39]
Where is https://github.com/expo/fyi/blob/master/expo-modules-core-migration.md (https://expo.fyi/expo-modules-core-migration) ?

Where is `react-native-unimodules` source code now?

**Comment by yfunk** [comment]
> We're in the process of migrating from `unimodules` packages to `expo-modules-core`, but for now it's happening only in the repo, so some things are not really published yet. The link to the migration guide doesn't work because we haven't written it yet — since some of those things are not fully migrated and we don't want to mislead you and others to migrate your apps too early 😉 We plan to finish all required work soon, so probably it'll be in time with the next SDK release (43), but to be honest we should make it in a week or so. By then, some files in the repo might be out of date — I'm really sorry for that, but there is no other way 😞. If you're looking for valid READMEs, I recommend using those that are on npm (that relates to versions you're using) or use https://docs.expo.dev
> 
> For now, keep using `react-native-unimodules` 🙂 We'll be posting updates on Twitter, once we officially release everything 😉

\- https://github.com/expo/expo/pull/14048#issuecomment-898948149

TLDR: The Expo team is currently in the process of migrating `react-native-unimodules` to `expo-modules-core`. None of these changes are published right now, so there isn't a migration guide yet and you can keep using `react-native-unimodules` for now.

The source code for `expo-modules-core` is [here](https://github.com/expo/expo/tree/master/packages/expo-modules-core).

**Comment by Bardiamist** [comment]
@yfunk thank you for the answer
